setup/express.js

```diff
@@ -0,0 +1,122 @@
+const expressWinston = require('express-winston');
+
+const { setupNodeLogger } = require('./winston');
+const { DEFAULT_LOG_LEVEL } = require('..');
+
+const defaultFilters = [
+  { agent: 'Consul Health Check', urls: ['/'], statuses: [200] },
+  { agent: 'Pingdom.com_bot_version', urls: ['/', '/api/v1/health'], statuses: [200] },
+];
+
+function skipRequests(filters) {
+  return (req, res) => {
+    const userAgent = req.headers['user-agent'] || '';
+
+    return filters.some(
+      ({ agent, urls, statuses }) => userAgent.includes(agent)
+        && urls.includes(req.url)
+        && statuses.includes(res.statusCode),
+    );
+  };
+}
+
+const whitelistedHeaders = [
+  'x-forwarded-for',
+  'x-forwarded-proto',
+  'content-type',
+];
+
+function ecsMeta(req, res) {
```

Comment:
What about - https://www.npmjs.com/package/@elastic/ecs-helpers ?
It has http formatter that does very similar thing - https://github.com/elastic/ecs-logging-js/blob/master/helpers/http-formatters.js
Is it different for us? Can we use it instead?

Reply:
yes, it's been discussed here already. It doesn't do what we need.

Reply:
I think I'd extract it to a separate file for it to be testable and add some tests for it.